src/SqlClient.Samples/WpfDataBinding/SqlServerTypes/readme.htm

```diff
@@ -0,0 +1,61 @@
+﻿<html lang="en-US">
```

Comment:
Could you clarify if this file comes from somewhere or if you made this adhoc?

Reply:
It appeared automatically. I think the source is because I opened/built the project with the latest VS2022.

Reply:
This actually came from Microsoft.SqlServer.Types update 12 -> 14, but actually we want to update it to latest version 160 because that supports .NET Standard. The file came with 14 but latest update did remove the file.